Assets/scss/components/quesionnaire.scss

```diff
@@ -0,0 +1,248 @@
+.cmp-questionnaire {
+  max-width: toRem(1175);
+  margin: auto;
+
+  @include max-screen(sm) {
+    max-width: toRem(767);
+  }
+
+  @include breakpoint-between(sm, md) {
+    max-width: toRem(1199);
+  }
+
+  &__container {
+    display: flex;
+    background-color: $grey94;
```

Comment:
Should this be a CSS variable instead of SCSS?

Reply:
Changed it to css variable.Thanks for the feedback.